Wikipedia/Code/Session.swift

```diff
@@ -697,3 +699,15 @@ class SessionDelegate: NSObject, URLSessionDelegate, URLSessionDataDelegate {
         callback.success()
     }
 }
+
+fileprivate extension URLRequest {
+    var prefersPersistentCacheOverError: Bool {
+        if let customCachePolicyValue = allHTTPHeaderFields?[URLRequest.customCachePolicyHeaderKey],
```

Comment:
Might be handy to implement the setter here too

Reply:
👍 Done!